src/sentry/integrations/slack/utils.py

```diff
@@ -313,8 +353,8 @@ def build_incident_attachment(incident):
                 },
             )
         ),
-        "text": " ",
-        "fields": fields,
+        "text": text,
+        "fields": [],
```

Comment:
Are you sure this will display the way you want? It could make sense to put some of these things as fields, rather than a concatenated query.